node/src/database/db_migrations/migrator_utils.rs

```diff
@@ -98,14 +101,14 @@ impl<'a> DBMigrationUtilities for DBMigrationUtilitiesReal<'a> {
 }
 
 struct DBMigDeclaratorReal<'a> {
```

Comment:
Maybe rename the name of the lifetime here, or remove it if possible.

Reply:
I checked that out, trying to knock some of the lifetimes off by using a Box instead but it started to be clear to me it wouldn't be trivial. I would like to do it either throughly or not at all, nothing between. So, I have an offer: if you do think this is an issue write a small card and maybe mark it as `good first issue`. Although the person will probably break his teeth at this meal. It's one file so he could get over it.